flent/runners.py

```diff
@@ -1006,6 +1006,13 @@ class SsRunner(ProcessRunner):
     time_re = re.compile(r"^Time: (?P<timestamp>\d+\.\d+)", re.MULTILINE)
     cwnd_re = re.compile(r"cwnd:(?P<cwnd>\d+)", re.MULTILINE)
     rtt_re = re.compile(r"rtt:(?P<rtt>\d+\.\d+/\d+\.\d+)", re.MULTILINE)
+
+    bbr_re_str = "bbr:\(bw:(?P<bw>\d+\.\d+)(?P<bw_scale>.bps),"\
+                 "mrtt:(?P<mrtt>\d+\.\d+),"\
+                 "pacing_gain:(?P<pacing_gain>\d+\.\d+),"\
+                 "cwnd_gain:(?P<cwnd_gain>\d+)\)"
```

Comment:
You are repeating these keys in three different places in the code. Use match.groupdict() instead to just pull them out of the regex match...